app/javascript/src/NewService/components/FormTemplate.jsx

```diff
@@ -27,13 +27,13 @@ const serviceDiscoveryProps: ServiceDiscoveryFormTemplateProps = {
     id: 'service_source'
   },
   serviceDiscoverySelect: {
-    namespace: {
+    project: {
```

Comment:
@guicassolato Sorry, I forgot to rename this.
I'm renaming only in the code, not in the UI, because of @thomasmaas comment about the need to update docs